### ISSUE TITLE
Enable ignoreFailures only for CI tasks

### ISSUE THREAD

**Issue by engi2nee** [JLLeitschuh/ktlint-gradle#408]
Currently, I am setting `ignoreFailures = false` so I can generate a ktlint report to use on Danger.
However, setting it to false also breaks the `pre-commit ` that is created using the `addKtlintCheckGitPreCommitHook` task, as the hook will ignore the failures and allows committing unformatted code.

is there is anyway I can set different values for this flag per task?

I tried using :
```
task runChecksForDanger() {
    group = "Reporting"

    subprojects.each { p ->
        p.ktlint {
            ignoreFailures = true
        }

        dependsOn 'ktlintAll'
    }}
```
But apparently it didn't work, as it only overrides the main ` ignoreFailures = false`

**Comment by Tapchicoma** [comment]
usually CI runners has `CI=true` env variable. Using it you could adjust `ignoreFailures` flag value.

**Comment by engi2nee** [comment]
Perfect, that worked out, don't know why I skipped this solution.
Thanks for your time
